sample-functions/python3/ab-test/index.py

```diff
@@ -46,7 +46,9 @@ def handler(request, context):
 
         filtered = list(filter(lambda host: host['weight'] > random, hosts))
 
-        redirect = filtered[0]['name'] + "/" + path
+        if path:
+            path = "/" + path
+        redirect = filtered[0]['name'] + path
```

Comment:
With this change you would also need to recompress this file to create an updated index.zip file.

Reply:
Holding off on zip pushes pending clarification from the team.